Using Carthage in a new project

I created a new Swift project, added a Cartfile with just Atlas in it and ran `carthage update`. I get several compilation errors, complaining that `fatal error: module 'LayerKit' not found`. Am I forgetting some obvious step?

@mmustapic Experiencing same bug:

Cartfile:

```ruby
github "layerhq/Atlas-iOS"
```

```bash
carthage update
```

```bash
*** Fetching Atlas-iOS
*** Fetching releases-ios
*** Downloading releases-ios.framework binary at "v0.19.4"
*** Checking out releases-ios at "v0.19.4"
*** Checking out Atlas-iOS at "v1.0.19"
*** xcodebuild output can be found in /var/folders/yv/zdbvrpv94kldv8ydn8c31zkr0000gn/T/carthage-xcodebuild.iPLb0k.log
*** Building scheme "Atlas" in Atlas.xcworkspace
** BUILD FAILED **


The following build commands failed:
	CompileC /Users/kevin/Library/Developer/Xcode/DerivedData/Atlas-cctztelohaayibakrsizwryptvto/Build/Intermediates/Atlas.build/Release-iphonesimulator/Atlas.build/Objects-normal/i386/Atlas.o Code/Atlas.m normal i386 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
/Users/kevin/Developer/Example/ios/Carthage/Checkouts/Atlas-iOS/Code/Utilities/ATLMessagingUtilities.h:22:9: fatal error: module 'LayerKit' not found
/Users/kevin/Developer/Example/ios/Carthage/Checkouts/Atlas-iOS/Code/Protocols/ATLMessagePresenting.h:23:9: fatal error: module 'LayerKit' not found
/Users/kevin/Developer/Example/ios/Carthage/Checkouts/Atlas-iOS/Code/Utilities/ATLMessagingUtilities.h:22:9: fatal error: module 'LayerKit' not found
/Users/kevin/Developer/Example/ios/Carthage/Checkouts/Atlas-iOS/Code/Controllers/ATLConversationListViewController.h:22:9: fatal error: module 'LayerKit' not found
A shell task failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	CompileC /Users/kevin/Library/Developer/Xcode/DerivedData/Atlas-cctztelohaayibakrsizwryptvto/Build/Intermediates/Atlas.build/Release-iphonesimulator/Atlas.build/Objects-normal/i386/Atlas.o Code/Atlas.m normal i386 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
```

--- a/Atlas.xcodeproj/project.pbxproj
+++ b/Atlas.xcodeproj/project.pbxproj
@@ -1429,7 +1429,7 @@
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
-					"$(PROJECT_DIR)/Carthage/Build/iOS",
+					"../releases-ios/**",
 				);
 				GCC_C_LANGUAGE_STANDARD = gnu99;
 				GCC_DYNAMIC_NO_PIC = NO;
@@ -1489,7 +1489,7 @@
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
-					"$(PROJECT_DIR)/Carthage/Build/iOS",
+					"../releases-ios/**",
 				);
 				GCC_C_LANGUAGE_STANDARD = gnu99;
 				GCC_NO_COMMON_BLOCKS = YES;

--- a/README.md
+++ b/README.md
@@ -124,7 +124,7 @@ Next bootstrap your environment by executing `carthage update`:
 $ carthage update
 ```
 
-Next drag `LayerKit.framework` and `Atlas.framework` from `Carthage/Build/iOS` onto your project and link it with your application target. Then select your application target within Xcode, navigate to the *Build Phases* panel and click the `+` icon and select *New Run Script Phase*. Set the content to:
+Next drag `Atlas.framework` from `Carthage/Build/iOS` and `LayerKit.framework` from `Carthage/Checkouts/releases-ios` onto your project and link it with your application target. Then select your application target within Xcode, navigate to the *Build Phases* panel and click the `+` icon and select *New Run Script Phase*. Set the content to:
 
 ```sh
 /usr/local/bin/carthage copy-frameworks
@@ -132,8 +132,8 @@ Next drag `LayerKit.framework` and `Atlas.framework` from `Carthage/Build/iOS` o
 
 In the *Input Files* section add:
 
-* $(SRCROOT)/Carthage/Build/iOS/LayerKit.framework
-* $(SRCROOT)/Carthage/Build/iOS/Atlas.framework
+* `$(SRCROOT)/Carthage/Checkouts/releases-ios/LayerKit.framework`
+* `$(SRCROOT)/Carthage/Build/iOS/Atlas.framework`
 
 Now build your application target and everything should be set.
 
@@ -199,7 +199,7 @@ The [ATLConversationViewController](Code/Controllers/ATLConversationViewControll
 
 #### Initialization
 
-```objc
+​```objc
 ATLConverationViewController *viewController = [ATLConversationViewController conversationViewControllerWithLayerClient:self.layerClient];
 ```
 

--- a/Resources/Info.plist
+++ b/Resources/Info.plist
@@ -20,5 +20,7 @@
 	<string>$(CURRENT_PROJECT_VERSION)</string>
 	<key>NSPrincipalClass</key>
 	<string></string>
+	<key>CFBundleExecutable</key>
+	<string>Atlas</string>
 </dict>
 </plist>

